docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -234,6 +234,13 @@ without renaming the entire workload. For example, specifying
 in a workload results directory named
 ``results-workload/DATE--TIME-super-application-test-v1/``.
 
+``compressresults``
+""""""""""""""""""""""""""
+
+Set this to ``yes`` if you want workload results to be compressed when copied back
```

Comment:
Consider making `compressresults` instead accept a compression algorithm as an argument (e.g.,  `compressresults=gzip`), which leaves open the option of supporting alternatives such as lz4 and zstd in the future.

Also, I'm curious about how the compression ratio compares to on-the-fly filesystem compression with zfs or btrfs, as opposed to ad-hoc support in FireSim.